### PR TITLE
:bug: Fix some layout errors

### PR DESCRIPTION
**Pull request by EvaMarco**
This PR solves layout erros

### CODE REVIEW

File: frontend/src/app/main/data/workspace/shape_layout.cljs [hunk 1]
```diff
@@ -82,22 +85,58 @@
...
+  (ptk/reify ::toogle-layout-flex
+    ptk/WatchEvent
+    (watch [_ state _]
+      (let [page-id          (:current-page-id state)
+            objects          (wsh/lookup-page-objects state page-id)
+            selected         (wsh/lookup-selected state)
+            selected-shapes  (map (d/getf objects) selected)
+            single?          (= (count selected-shapes) 1)
+            is-frame?        (and single? (= :frame (:type (first selected-shapes))))
+            undo-id          (uuid/next)
```
Comment (Alotor) [inline frontend/src/app/main/data/workspace/shape_layout.cljs]: Remove the end of line here and in the the other two functions calls bellow

File: frontend/src/app/main/data/workspace/shape_layout.cljs [hunk 2]
```diff
@@ -109,14 +148,11 @@
             selected         (wsh/lookup-selected state)
             selected-shapes  (map (d/getf objects) selected)
             single?          (= (count selected-shapes) 1)
-            has-flex-layout? (and single? (= :flex (:layout (first selected-shapes))))
-            is-frame?        (and single? (= :frame (:type (first selected-shapes))))]
+            has-flex-layout? (and single? (= :flex (:layout (first selected-shapes))))]
```
Comment (Alotor) [inline frontend/src/app/main/data/workspace/shape_layout.cljs]: you can use the function `layout?` in `app.common.types.shape.layout`

File: frontend/src/app/main/data/workspace/shapes.cljs [hunk 3]
```diff
@@ -362,12 +362,44 @@
...
-              (dwu/start-undo-transaction)
+              (dwu/start-undo-transaction undo-id)
               (add-shape shape)
               (move-shapes-into-frame (:id shape) selected)
-              (dwu/commit-undo-transaction)))))))))
+              (dwu/commit-undo-transaction undo-id)))))))))
+
+
+
+(defn create-artboard-from-selection-2
```
Comment (Alotor) [inline frontend/src/app/main/data/workspace/shapes.cljs]: This is no longer necesary, isn't it?

Comment (EvaMarco) [reply frontend/src/app/main/data/workspace/shapes.cljs]: ups, yes¡¡

**Pull request merged**